conn/ssl.h

```diff
@@ -28,6 +28,11 @@ struct Connection;
 
 int mutt_ssl_starttls(struct Connection *conn);
 int mutt_ssl_socket_setup(struct Connection *conn);
+#else
+static inline int mutt_ssl_socket_setup(struct Connection *conn)
```

Comment:
Uhm.. so the forward definition of `struct Connection` above is needed in all cases.

Reply:
Right. I guess I didn't get bitten by that because it was already done by another header. I'll fix that.